js/components/LeftNavArrow/styles.js

```diff
@@ -0,0 +1,17 @@
+import { StyleSheet } from 'react-native';
+
+export const styles = StyleSheet.create({
+  container: {
+
```

Comment:
Fix extra whitespace.

Reply:
Fixed white space fro Left and Right Nav Arrow